include/aos/common/pkcs11/privatekey.hpp

```diff
@@ -141,6 +141,11 @@ class PKCS11ECDSAPrivateKey : public crypto::PrivateKeyItf {
     crypto::ECDSAPublicKey     mPublicKey;
 };
 
+/**
+ * Maximum size of pkcs11 PrivateKey in bytes.
+ */
+constexpr auto cPrivateKeyMaxSize = Max(sizeof(PKCS11RSAPrivateKey), sizeof(PKCS11ECDSAPrivateKey));
```

Comment:
So far it used only by crypto utils. Could we make it private in CertLoader?

Reply:
I introduced it to pkcs11 module also